### PR TITLE
Pstuffa/cleanup

### PR DESCRIPTION
**Pull request by pstuffa**

### CODE REVIEW

File: docs/accounts-workspaces/accounts.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ An Observable account makes it possible for you to use the Observable platform a
 To get started, click the **Sign up** button on https://www.observablehq.com.
 
 :::info Note
-If you <a href="https://observablehq.com/@observablehq/fork-suggest-merge">fork</a> a notebook before you have created an account, you will be asked to sign up to save your fork; the dialog box will look very similar to the one below.`)
+If you fork a notebook before you have created an account, you will be asked to sign up to save your fork; the dialog box will look very similar to the one below.`)
```
Comment (eagereyes) [inline docs/accounts-workspaces/accounts.md:10]: There's a spurious backtick and closing parenthesis at the end here

File: docs/cells/index.md [hunk 2]
```diff
@@ -95,6 +94,10 @@ Cell suggestions are over to the right side of the space that the next cell's **
 
 Some suggestions create a new cell below in an effort to keep you moving to the next logical step in the analysis process (e.g. the **Create chart** suggestion in the screenshot above). Suggestions that show an arrow indicate that the focused cell will be non-destructively transformed into another cell (e.g. the **Convert to SQL** suggestion in the screenshot above).
 
+## Evaluating cells
+
+To have the results of your cell appear, click the Play (▶︎) button, or type <Keys set="Shift-Enter" /> to run your cell.
```
Comment (eagereyes) [inline docs/cells/index.md:99]: play icon should be <Icon name="playButton" /> :)

File: docs/cells/data-table.md [hunk 3]
```diff
@@ -463,7 +463,7 @@ The summary charts are an extension of previous work on the [Summary Table](http
 
 ## Convert to SQL
 
-If you want to create a hand-tuned SQL query on selected results, you can convert a populated data table cell to a [SQL cell](https://observablehq.com/@observablehq/sql). Select the **+** below a data table cell and select **Convert to SQL**.
+If you want to create a hand-tuned SQL query on selected results, you can convert a populated data table cell to a [SQL cell](/cells/sql-cell). Select the **+** below a data table cell and select **Convert to SQL**.
```
Comment (eagereyes) [inline docs/cells/data-table.md:466]: After my rename, it's now `/cells/sql`

File: docs/debugging/minimap.md [hunk 4]
```diff
@@ -54,16 +54,16 @@ When one cell is focused, the Minimap draws connections to other cells in blue:
 
 ## Using keyboard shortcuts in the Minimap
 
-The Minimap has no special keyboard shortcuts of its own, but works well with the [existing shortcuts](https://observablehq.com/@observablehq/keyboard-shortcuts?collection=@observablehq/notebook-fundamentals) for moving among cells, including:
+The Minimap has no special keyboard shortcuts of its own, but works well with the [existing shortcuts](/misc/keyboard-shortcutss) for moving among cells, including:
```
Comment (eagereyes) [inline docs/debugging/minimap.md:57]: There's an extra s at the end of the URL

File: docs/embeds/troubleshooting.md [hunk 5]
```diff
@@ -54,71 +57,71 @@ The Iframe works most consistently in the most places; the two Runtime methods o
 - **WordPress.org (self-hosted)**. You can paste either the Iframe code or the Runtime with JavaScript code into a Custom HTML block. Note that the Runtime with JavaScript option may introduce some issues with the width of the embed in some cases (see “The width is wrong” below), so you may want to stick to the Iframe code. [More info](https://wordpress.org/plugins/iframe/)
 - **Other**. What we’re calling “embedding” here does not cover embedding in social media sites, which generally don’t support interactive embedding; there, we use the notebook’s title, author, description, and thumbnail. It also doesn’t cover embedding in most productivity software, which generally don’t let you run code. You can download a version of most graphical cells by clicking “Download PNG” in the cell menu.
 
-## The cell I want to embed doesn’t appear in the list
+### The cell I want to embed doesn’t appear in the list
```
Comment (eagereyes) [inline docs/embeds/troubleshooting.md:60]: I actually changed all of these to H2s to show up in the navigation overview at the top, so you quickly find them without scrolling through the entire document

File: docs/notebooks/templates.md [hunk 6]
```diff
@@ -4,7 +4,7 @@ Templates provide pre-packaged artifacts for common data-related workflows. Whet
 
 This page describes how you can use existing templates to avoid starting your work from scratch, and how to make your own custom templates for reuse by you and your team.
 
-## Use existing templates {#existing-templates}
+## Observable templates
 
 Observable provides a number of [pre-made templates](http://observablehq.com/templates) for common data-related tasks like boilerplates for basic charts and maps, starters to access APIs or connect to Google sheets, and templates for common analyses for business planning and product insights.
```
Comment (eagereyes) [inline docs/notebooks/templates.md:9]: This should be https:// – we might want to do a global search for `http://observ` to fix all of these

**Review by eagereyes — APPROVED** [review]
I flagged a few minor things, rest looks great!

File: docs/embeds/troubleshooting.md (hunk 5 above)
Comment (pstuffa) [reply docs/embeds/troubleshooting.md:60]: The problem is that they're too long to be read in the TOC. You just get "The cell I want..." I think most users will (hopefully) just use search to find these.

**Pull request merged**